backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveDiskCommand.java

```diff
@@ -170,11 +170,15 @@ protected Map<String, Pair<String, String>> getExclusiveLocks() {
 
     private String getDiskIsBeingMigratedMessage() {
         if (cachedDiskIsBeingMigratedMessage == null) {
+            DiskImage diskImage = diskImageDao.get(getParameters().getImageId());
+            String diskName = "";
+            if (diskImage != null) {
+                diskName = diskImage.getDiskAlias() != null ?  diskImage.getDiskAlias() : "";
+            }
             cachedDiskIsBeingMigratedMessage = new LockMessage(EngineMessage.ACTION_TYPE_FAILED_DISK_IS_BEING_MIGRATED)
-                    .withOptional("DiskName", getParameters().getNewAlias())
+                    .withOptional("DiskName", diskName)
```

Comment:
AFAIU, since you are using `withOptional()` (that can handle `null`), lines 174-177 can be a simple:
`String diskName = (diskImage == null) ? null : diskImage.getDiskAlias();`